### PR TITLE
add docs for smartbft metrics

### PR DESCRIPTION
**Pull request by pfi79**

### CODE REVIEW

**Comment by pfi79** [comment]
Can anyone reset the piplines for sbe nwo msp?

File: common/metrics/gendoc/options.go [hunk 1]
```diff
@@ -72,7 +72,8 @@ func FileOptions(f *ast.File) ([]interface{}, error) {
 				if !ok {
 					continue
 				}
-				if imports[ident.Name] != "github.com/hyperledger/fabric/common/metrics" {
```
Comment (yeasy) [inline common/metrics/gendoc/options.go:75]: This is not only doc change, but some code change too.

Comment (pfi79) [reply common/metrics/gendoc/options.go:75]: yes, without this change, there would be no documentation

**Comment by pfi79** [comment]
@yacovm, perhaps you might have time to review pr properly?

**Review by yacovm — APPROVED** [review]

**Pull request merged**